packages/driver/src/cy/mouse.js

```diff
@@ -404,9 +408,14 @@ const create = (state, keyboard, focused) => {
 
       const mouseUpEvents = mouse.up(fromElViewport, forceEl, skipMouseupEvent, pointerEvtOptionsExtend, mouseEvtOptionsExtend)
 
-      const skipClickEvent = $elements.isDetachedEl(mouseDownEvents.pointerdownProps.el)
+      // Only send click event if the same element received both pointerdown and pointerup, and it's not detached.
+      // This will always be false for force click, so short circuit it here with !forceEl
+      const skipClickEvent = !forceEl && (
+        ($elements.isDetachedEl(mouseDownEvents.pointerdownProps.el) && 'Element was detached') ||
```

Comment:
Where does this string `Element was detached` come from? Won't a string always evaluate to true?

Ok, read down to the caller. This is a strange way to me to return an error message. I would rather this be some kind of explicit throw, otherwise I'm sitting here like....what the hell is this string in this comparison operators for? - without the context of the caller this is really weird.

Reply:
true, I'll clean this up

Reply:
So much clearer. 😅